content/recipes/introduction/priorization.md

```diff
@@ -0,0 +1,207 @@
+
+**Hence, the best time of engaging a project into the FAIR pipeline would be dependent on the retrospective and prospective aspect of the project.** The best time to FAIRify prospective projects is the early stage. This stage is the best time to define and lay out metadata standards that need to be followed during the project trajectory. As a result, it eliminates many of the downstream logistical and financial problems. On the other hand, the FAIRification of retrospective projects is in the middle stage. This is because it is a data rich stage and contact with both data owners and generators could be established for better understanding of the data.
+
+#### 3. Prioritisation based on partners
+	
+A project has the potential to involve a large number of people each coming from various institutions. Consequently, users can also consider the wider consortia involved as a criteria for prioritising projects from a list of projects. In the end, the FAIRification process should benefit as many people as possible.
+
+**Thus, projects that have diverse partners involved (e.g.  academic, industrial, start-up, and so on)  should be prioritised over singleton partners, that is those projects that involve people from the same institute or industrial group.** The main reason for this priority is the impact of the FAIRification process. Within an intra-organization group, the data generators, maintainers, and depositors might have the same terminology and definitions related to the data and hence, a FAIRification process would only be needed when the data has to be deposited on public repositories or databases. On the other hand, in an inter-organization group definitions and terminologies are not consistent, and the transfer of data from data generators to depositors might be time-consuming. As a result, the FAIRification process would benefit by increasing transparency between intra-organizational groups.
+
+#### 4. Prioritisation based on existence of Data Management Plan (DMP)
+
+A data management plan (DMP) is a document that describes the life cycle of a data beginning from its generation, followed by processing and collection, then dissemination, and finally the usage (More details in [Data Management Plan]() recipe). This established document gives the data owners and FAIR experts an overview of the resource under study. On the basis of this DMP document, the FAIRification process can be determined and established. In case of absence of the DMP, the FAIR experts lack the ability to find the best FAIRification process for the given data thereby leading to inefficient results. Overall, the DMP plan would potentially point to the data regulatory aspect considering the data usage as well as dessiementation of the data into public resources. **Thus, projects that have their DMP documents established should be prioritised over those projects that do not have one ready yet.**
```

Comment:
```suggestion
A data management plan (DMP) is a document that describes the life cycle of a data beginning from its generation, followed by processing and collection, then dissemination, and finally the usage (More details in [Data Management Plan]() recipe). This established document gives the data owners and FAIR experts an overview of the resource under study. On the basis of this DMP document, a FAIRification process can be determined and established. In case of the absence of a DMP, the FAIR experts lack the ability to find the best FAIRification process for the given data thereby leading to inefficient results. Overall, the DMP plan would potentially point to the data regulatory aspect, considering the data usage as well as dissemination of the data into public resources. **Thus, projects that have their DMP documents established should be prioritized over those projects that do not have one ready yet.**
```